app/controllers/users_controller.rb

```diff
@@ -1,4 +1,31 @@
 class UsersController < ApplicationController
   def new
+    @user = User.new
   end
+
+  def show
+    @user = User.find_by id: params[:id]
```

Comment:
use call back for this block code
please refer more at:
https://viblo.asia/LeAnhTuan/posts/MLzkOYQRGpq
http://guides.rubyonrails.org/active_record_callbacks.html